apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -18,57 +20,28 @@ export async function createProject({
   resource,
   store = resourceStore(),
   user,
-}: CreateProjectCommand): Promise<GraphPointer> {
-  const label = resource.out(rdfs.label).term!
-
-  const project = await store
-    .createMember(projectsCollection, id.cubeProject(label.value))
-
-  project.addOut(rdfs.label, label)
-    .addOut(dcterms.creator, user)
-
-  if (shape('cube-project/create#CSV').equals(resource.out(cc.projectSourceKind).term)) {
-    const mapping = store
-      .create(id.csvMapping(project))
-      .addOut(rdf.type, cc.CsvMapping)
+}: CreateProjectCommand): Promise<Project.Project> {
```

Comment:
Now the code to create a new project is split between the handler, this function and the mixin classes. Is there a reasoning to know what should go where?